prov/rxm/src/rxm.h

```diff
@@ -189,6 +191,7 @@ struct rxm_cmap_handle {
 	uint64_t remote_key;
 	fi_addr_t fi_addr;
 	struct rxm_cmap_peer *peer;
+	struct slist_entry wait_conn_quota_entry;
```

Comment:
nit: align name with the name of the list -- e.g. conn_wait_quota_entry.  Though I'd prefer these be shorted to just conn_wait_entry and conn_wait_list